modules/service-account/outputs.tf

```diff
@@ -1,3 +1,27 @@
+output "primary_cluster" {
```

Comment:
My question here is if an end user looking at this output in a module will have the context on what this is, as `module.service-account.primary_cluster` to me is somewhat ambiguous around if it is referencing separate clusters entirely.

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/54/commits/16f7c3fba5e9d24ed16d66ffd9d78183072bf957.